c2rust-analyze/tests/filecheck/foreign.rs

```diff
@@ -0,0 +1,25 @@
+extern "Rust" {
```

Comment:
Don't we want to mainly test `extern "C"`?

Reply:
Can i have that and still put the Rust types as arguments? I didnt think one type of extern block or another would make a difference for this feature

Reply:
aa23b35d

Reply:
Can you make `Bar` and `Baz` `#[repr(C)]`?

Reply:
they are in 9f35bbf